cmd/skopeo/sync.go

```diff
@@ -723,6 +726,19 @@ func (opts *syncOptions) run(args []string, stdout io.Writer) (retErr error) {
 		logrus.Warn("Running in dry-run mode")
 	}
 
+	var digestFile *os.File
+	if opts.digestFile != "" && !opts.dryRun {
+		digestFile, err = os.OpenFile(opts.digestFile, os.O_CREATE|os.O_WRONLY, 0644)
+		if err != nil {
+			if os.IsNotExist(err) {
+				return fmt.Errorf("File system is likely out of inodes or has other issues.: %w", err)
```

Comment:
Why should “IsNotExist” map to “out of inodes???” That’s by far not the most common cause (almost certainly be a typo in a parent directory name).

Why does this need a special case for `IsNotExist` anyway?